python/pyxet/pyxet/util.py

```diff
@@ -31,16 +31,24 @@ def _should_load_aws_credentials():
         return False
     return True
 
-
 def _get_fs_and_path(uri, strip_trailing_slash = True):
```

Comment:
Can we get rid of this debug wrapper?

Reply:
We should, but it's there for historical reasons so let's do that in a different PR.

Reply:
sounds good